src/py42/util.py

```diff
@@ -3,9 +3,10 @@
 import json
 from datetime import datetime
 
-from py42._compat import str
+from py42._compat import string_type
 
-_MICROSECOND_FORMAT = u"%Y-%m-%dT%H:%M:%S.%fZ"
+MILLISECOND_FORMAT = u"%Y-%m-%dT%H:%M:%S.%f"
+MICROSECOND_FORMAT = u"%Y-%m-%dT%H:%M:%S.%fZ"
```

Comment:
My understanding is that the `Z` signifies that the timestamp is in UTC, rather than suggesting anything about the precision of the seconds value.

Reply:
Yeah it is confusingly being used here...
The millisecond one truncates itself in the places it gets used

Reply:
I can try and find a way to reword this or re-do something somewhere

Reply:
Just pushed something ... does this wok? I am a little nervous changing this now.

Reply:
Yeah, I think that should work fine